index.html

```diff
@@ -12,13 +12,24 @@
 </head>
 
 <body>
-    <div id="app">
+<div id="app">
```

Comment:
Again, html files follows a convention that is better to be followed (tabs and spaces..), you can automatically "fix" that with js-beautify, like before